js/dev-server.js

```diff
@@ -8,7 +8,7 @@ const port = process.env.PORT || '4000';
 
 // Serve static files from the Jekyll build
 const static_path = path.resolve(path.join(__dirname), '..', '_site');
-
+const assets_path = path.resolve(path.join(__dirname), '..', 'www.foia.gov', 'assets');
```

Comment:
@mrclay Can you help me understand this part? Since Jekyll compiles everything -- including that assets folder -- into _site, why does the dev server need to host that separately? Is it because the dev server can't do subfolders of _site?

Reply:
I think I mistakenly thought one should be able to `make serve` with no jekyll build in place. I suppose you could have `serve` depend on `build` to make that clear. Or I'll just wait till I'm more familiar with the project :)

Reply:
To be clear: if jekyll output should always be present during `serve`, then we should not add this assets_path.